src/arbcall.jl

```diff
@@ -100,24 +99,47 @@ function jlcode(af::Arbfunction, jl_fname=jlfname(af))
     if !isnothing(k)
         @assert jl_types[k] == arb_rnd
         r = :rnd
-        push!(kwargs, Expr(:kw, :($r::Union{arb_rnd, RoundingMode}), :(RoundNearest)))
+        push!(jl_kwargs, Expr(:kw, :($r::Union{arb_rnd, RoundingMode}), :(RoundNearest)))
         deleteat!(arg_names, k)
         deleteat!(jl_types, k)
     end
 
     jl_args = [:($a::$T) for (a, T) in zip(arg_names, jl_types)]
 
-    res = first(arg_names)
+    :($(jlfname(af))($(jl_args...); $(jl_kwargs...))::$(returnT))
```

Comment:
to be honest it is not clear to me what is the actual benefit of `jlsignature` (except maybe testing?)
to make both testing and the generated code reading easier I suggest that the function just returns a tuple
```julia
return (jl_args, jl_kwargs)
```
note that `returnT` and `jlfname` can and should be computed directly from `af::Arbfunction` and not tangled into another function, when the're directly accessible outside it;
maybe renaming the function to `jlargs(af::Arbfunction)` also makes sense?

Reply:
I must admit I mainly wrote it for testing purposes, and then thought it would be natural to use it in `jlcode` as well. But I agree, to much information is mangled into this one method. I can change it to `jlargs` as you propose!

Reply:
Now I have rewritten it in terms of `jlargs`. This does feel cleaner!